airbyte-integrations/connectors/destination-local-json/src/main/resources/spec.json

```diff
@@ -11,7 +11,8 @@
       "destination_path": {
         "description": "Path to the directory where json files will be written. The files will be placed inside that local mount. For more information check out our <a href=\"https://docs.airbyte.io/integrations/destinations/local-json\">docs</a>",
         "type": "string",
-        "examples": ["/json_data"]
+        "examples": ["/json_data"],
+        "pattern": "(^\\/json_data\\/.*)|(^\\/json_data$)"
```

Comment:
@cgardens, is it OK that the destination code checks for `/local` but not `/json_data` ?

https://github.com/airbytehq/airbyte/blob/9ec0ba91400e9229bf11b6b55fb688ead14fe72e/airbyte-integrations/connectors/destination-local-json/src/main/java/io/airbyte/integrations/destination/local_json/LocalJsonDestination.java#L130

Reply:
i don't have context into that decision on the naming here. i just know that the input for this destination is supposed to be `/json_data` and the UI was not validating that. this PR makes sure it does.